core/include/cib.h

```diff
@@ -44,9 +46,9 @@ typedef struct {
  *
  * @param[out] cib      Buffer to initialize.
  *                      Must not be NULL.
- * @param[in]  size     Size of the buffer, must not exceed MAXINT/2.
+ * @param[in]  size     Size of the buffer, must not exceed 65,534.
```

Comment:
no, 65535 would be the correct value here.2^16 = 65536 and so uint16_t is from 0 to 65535, so 65535 would be the maximum value, right?